src/lib/deprecated-input-normalizers.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/* Returns values from the v1 'user' key.  Warns in the event a 'deprecated' value is passed */
+export const setUserNormalizer = (input : any) => {
+    if (input) {
+        if (input.user) {
+            // eslint-disable-next-line no-console
+            console.warn('the "user" key has been deprecated. See v2 documentation for details');
+            return input.user;
+        }
+    }
+
+    return input;
+};
+
+/* sets cartTotal to total. Warns in the event a 'deprecated' value is passed */
+export const addToCartNormalizer = (input : any) => {
+    if (input) {
+        if (input.total !== undefined) {
+            // eslint-disable-next-line no-console
+            console.warn('"total" has been deprecated. use "cartTotal" instead');
+        } else if (input.cartTotal) {
+            input.total = input.cartTotal;
```

Comment:
Why not `input.cartTotal = input.total`? I would assume we should be sending the new keys to the back-end, not the old ones (basically looks like you're normalizing v2 into v1 here, instead of v1 into v2).

Reply:
Had an offline discussion -- we're using "total" on the back-end, and changing that would require a change to multiple services and the database. Max said he'll put a code comment here explaining that.